package/msys/magnum-plugins/PKGBUILD

```diff
@@ -0,0 +1,71 @@
+# Author: williamjcm <w.jcm59@gmail.com>
+# Contributor: mosra <mosra@centrum.cz>
+# Based on the magnum-plugins Arch Linux PKGBUILD maintained by xyproto and the MSYS2/MinGW PKGBUILD templates
+
+_realname=magnum-plugins
+pkgbase=mingw-w64-${_realname}
+pkgname="${MINGW_PACKAGE_PREFIX}-${_realname}"
+pkgver=2018.10
+pkgrel=1
+pkgdesc='Plugins for the Magnum C++11/C++14 graphics engine'
+arch=('any')
+url='https://magnum.graphics/'
+license=('MIT')
+depends=("${MINGW_PACKAGE_PREFIX}-magnum>=$pkgver"
+         "${MINGW_PACKAGE_PREFIX}-assimp"
+         "${MINGW_PACKAGE_PREFIX}-devil"
+         "${MINGW_PACKAGE_PREFIX}-freetype"
+         "${MINGW_PACKAGE_PREFIX}-libjpeg-turbo"
+         "${MINGW_PACKAGE_PREFIX}-libpng")
+makedepends=("${MINGW_PACKAGE_PREFIX}-cmake" "${MINGW_PACKAGE_PREFIX}-ninja" 'git')
+# The .tar.gz / .zip download contains symlinks (.travis.yml), making tar and
+# unzip on Windows grumpy because dangling symlinks are created. Could be fixed
+# by manually extracting everything except symlinks, but that's hard to
+# maintain. Downloading a Git tag works.
+source=("${_realname}-${pkgver}"::"git+https://github.com/mosra/magnum-plugins.git#tag=v$pkgver")
+sha1sums=('SKIP')
+
+build() {
+    mkdir -p "${srcdir}"/build-${CARCH} && cd "${srcdir}"/build-${CARCH}
+
+    declare -a extra_config
+    if check_option "debug" "n"; then
+        extra_config+=("-DCMAKE_BUILD_TYPE=Release")
+    else
+        extra_config+=("-DCMAKE_BUILD_TYPE=Debug")
+    fi
+    
+    MSYS2_ARG_CONV_EXCL="-DCMAKE_INSTALL_PREFIX=" \
+        ${MINGW_PREFIX}/bin/cmake \
+            -G'Ninja' \
+            -DCMAKE_INSTALL_PREFIX=${MINGW_PREFIX} \
+            -DWITH_ASSIMPIMPORTER=OFF \
+            -DWITH_DDSIMPORTER=ON \
+            -DWITH_DEVILIMAGEIMPORTER=ON \
+            -DWITH_DRFLACAUDIOIMPORTER=ON \
+            -DWITH_DRWAVAUDIOIMPORTER=ON \
+            -DWITH_FREETYPEFONT=ON \
```

Comment:
Oh, interesting, HarfBuzz has a MSYS package too? Then I would do `WITH_HARFBUZZFONT=ON` here as well (and the corresponding dependency above).

The reason I did this in the Arch PKGBUILD was originally to reduce the amount of dependencies, but harfbuzz is pretty ubiquitous nowadays, so I think I'll enable it on Arch as well.

Reply:
> HarfBuzz has a MSYS package too?

Yeah, it does. It's possible I mentioned it on Gitter months ago, back when I started with Magnum, but I could be wrong.